src/main/java/apoc/export/cypher/formatter/CypherFormatterUtils.java

```diff
@@ -0,0 +1,199 @@
+package apoc.export.cypher.formatter;
+
+import apoc.export.util.FormatUtils;
+import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.Node;
+import org.neo4j.helpers.collection.Iterables;
+
+import java.lang.reflect.Array;
+import java.util.*;
+
+/**
+ * @author AgileLARUS
+ *
+ * @since 16-06-2017
+ */
+public class CypherFormatterUtils {
+
+	public final static String UNIQUE_ID_LABEL = "UNIQUE IMPORT LABEL";
+	public final static String UNIQUE_ID_PROP = "UNIQUE IMPORT ID";
+	public final static String Q_UNIQUE_ID_LABEL = quote(UNIQUE_ID_LABEL);
+
+    public static class NodeInfo {
+        private Set<String> nodeIdLabels = new HashSet<>();
```

Comment:
@jexp you're right. we should avoid storing data in this helper inner class.
Nevertheless I would keep it as the 4 formatters benefit a lot from it in term of code cleanness.

What if we remove all fields, make all methods static and move into them the logic currently in the constructor?